### ISSUE TITLE
At what point have we implemented Logistic Regression? 

### ISSUE THREAD

**Issue by nirdesh80** [ushasi/MachineLearning#1]
Hi @ushasi !

Thank you for sharing the repository and keeping is updated.

I was going through the code for Logistic Regression > C-MInDS-MLCourse/Logistic Regression/logistic_regression_gd.ipynb

I could follow the part where Gradient Descent and Loss function get evaluated
but I failed to understand at what stage did we implement the fit function for logistic regression?
I mean I was expecting something of the sort of clf = LogisticRegression().fit(X_train, y_train)

~ Niranjan

**Comment by ushasi** [comment]
Hi Niranjan,
The code that you are looking at implements Logistic Regression from scratch. So there are no such inbuilt functions like  LogisticRegression() or fit(X_train, y_train). For this purpose, you can look into the code Logistic Regression/logistic_regression.ipynb which uses SkLearn library functions like fit() and LogisticRegression().

Thanks,
